src/components/Modal.js

```diff
@@ -60,16 +61,36 @@ const Modal = (props) => {
             <SafeAreaInsetsContext.Consumer>
                 {(insets) => {
                     const {paddingTop, paddingBottom} = getSafeAreaPadding(insets);
+
+                    /**
+                     * Calculates the real top and bottom padding of the container,
+                     * given the given padding, the plataform specific safe area padding,
+                     * and if it should sum both values or not
+                     *
+                     * @param {number} containerPadding
+                     * @param {number} safeAreaPadding
+                     * @param {boolean} shouldSumSafeAreaPadding
+                     * @returns {number}
```

Comment:
We have standardized on doc params to be uppercase like `{Number}`

Reply:
Noted! 
But removed the function.